.github/workflows/integration-test.yaml

```diff
@@ -65,10 +65,46 @@ jobs:
         group: ${{ fromJson(needs.group-diff.outputs.groups) }}
         experimental: [true]
       fail-fast: false
-    uses: intel/ai-containers/.github/workflows/container-ci.yaml@main
+    uses: ./.github/workflows/container-ci.yaml
```

Comment:
And here

Reply:
Will make the changes after I have done revalidating.